autoextract_spiders/settings.py

```diff
@@ -8,6 +8,7 @@
 #     http://doc.scrapy.org/en/latest/topics/settings.html
 #     http://scrapy.readthedocs.org/en/latest/topics/downloader-middleware.html
 #     http://scrapy.readthedocs.org/en/latest/topics/spider-middleware.html
+import autoextract_spiders
```

Comment:
Is there any implicit reason to add it here?

Reply:
Ops. Removed here: https://github.com/scrapinghub/autoextract-spiders/pull/25/commits/0ed04ba41c109231a64f448d02579137421936bc